lib/blueprinter/base.rb

```diff
@@ -274,6 +275,43 @@ def self.fields(*field_names)
       end
     end
 
+
+
+    # Specify one transformer to be included for serialization.
+    # Takes a class which extends Blueprinter::Transformer
+    #
+    # @param class name [Class] which implements the method transform to include for
+    #   serialization.
+    #
+    #    
+    # @example Specifying a DynamicFieldTransformer transformer for including dynamic fields to be serialized.
+    #   class User 
+    #      def custom_columns
+    #         self.dynamic_fields #which is an array of some columns
+    #      end
+    #      def custom_fields
+    #         custom_columns.each_with_object({}){|col,result|  result[col] = self.send(col)}
+    #      end
+    #   end
+    #
+    #   class UserBlueprint < Blueprinter::Base
+    #     fields :first_name, :last_name
+    #     transform DynamicTransformer  
+    #     # other code
+    #   end
+    #
+    #   class DynamicFieldTransformer < Blueprinter::Transformer
```

Comment:
Should this be `DynamicTransformer`? (or `transform DynamicFieldTransformer` above)?

Reply:
Yup, example mismatch. would appreciate if you can make a PR.

Reply:
Done, see #174